### ISSUE TITLE
Consider changing search drop-downs to check box selects

### ISSUE THREAD

**Issue by elimisteve** [lucyparsons/OpenOversight#262]
At https://openoversight.lucyparsonslabs.com/find in particular, doesn't it make sense for Officer Rank and maybe Race to be checkboxes, so that more than one can be selected?  Being able to effectively say, "I don't know what his rank was", or, "he was Asian or Pacific Islander" is nicer than having to do multiple separate queries.

And I assume lots of people will be like me and not have any idea what the officer's rank is, so maybe still defaulting to Commander makes sense, but also having a "Check all" or "I don't know" button that checks all would be good, or defaulting to all.

**Comment by redshiftzero** [comment]
Yep I agree with dumping the dropdown. That's similar to one of the feedback points we got in our [heuristic UX analysis](https://github.com/lucyparsons/OpenOversight/issues/252#issue-226799014) - check that out for a little screenshot of an example form without dropdowns. 

We'd also need to be sure we can handle filtering by multiple ranks and races in the query (right now it assumes just one of each in `filter_by_form` in `utils.py`).

You can currently select "Don't know" in order not to filter by that attribute but that option should definitely be more clear.
